src/cursor/cur_ds.c

```diff
@@ -425,6 +425,7 @@ __wt_curds_open(WT_SESSION_IMPL *session, const char *uri, WT_CURSOR *owner, con
       __curds_remove,                                 /* remove */
       __curds_reserve,                                /* reserve */
       __wt_cursor_reconfigure_notsup,                 /* reconfigure */
+      __wt_cursor_largest_key,                        /* largest_key */
```

Comment:
I'm not sure what is a data source cursor.

Reply:
A data source cursor is an application-specific cursor, where an application has its own data source. I'd just not support this API in that case.